modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/styles/_variables.scss

```diff
@@ -12,6 +12,8 @@ $info-primary: #2e5aac;
 
 $error-l1: lighten($danger-primary, 28%);
 $gray-600: #6b6c7e;
+$gray-100: #f7f8f9;
+$table-text: #272833;
```

Comment:
If it's decided to `@import 'atlas-variables'`, we can remove this and use `$table-list-color` directly.

Reply:
That would be indeed desirable and in line with what we're doing in other places already!